### PR TITLE
ibc v5

### PR DESCRIPTION
**Pull request by faddat**
This is the interchain accounts demo, but it uses ibc v5 and cosmos-sdk v0.46.0

- ibcv5
- ibcv5 add testnet commands

### CODE REVIEW

**Review by seantking — APPROVED** [review]
:rocket:

**Review by damiannolan — APPROVED** [review]
Excellent thanks, can you confirm you ran this locally successfully?

Appreciate all the work on the `v0.46` upgrades

**Comment by faddat** [comment]
I can confirm that it passed all tests-- is there anything more you'd like me to try?

File: app/app.go [hunk 1]
```diff
@@ -297,7 +299,7 @@ func New(
 
 	// add keepers
 	app.AccountKeeper = authkeeper.NewAccountKeeper(
-		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms,
+		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms, sdk.Bech32MainPrefix,
```
Comment (crodriguezvega) [inline app/app.go:302]:
```suggestion
		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms, sdk.GetConfig().GetBech32AccountAddrPrefix(),
```

**Review by crodriguezvega — COMMENTED** [review]
Thank you, @faddat! I just left a couple of nits.

**Comment by seantking** [comment]
Merged. Many thanks! @faddat

**Pull request merged**